docs/streams/upgrade-guide.html

```diff
@@ -191,6 +191,17 @@ <h3><a id="streams_api_changes_210" href="#streams_api_changes_210">Streams API
         details, see <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-312%3A+Add+Overloaded+StreamsBuilder+Build+Method+to+Accept+java.util.Properties">KIP-312</a>
     </p>
 
+    <p>
+        We are introducing static membership towards KStream users. This feature reduces unnecessary rebalances during normal application upgrades or rolling bounces.
+        If you want to use static membership, first thing is to upgrade both broker cluster and stream apps to 2.3.0. This involves a necessity to change <code>inter.broker.protocol.version</code>
+        to 2.3 as well. Then you need to set the config <code>ConsumerConfig#GROUP_INSTANCE_ID_CONFIG</code> to a universally unique value for each stream instance.
+        If your broker is on older version, but you choose to set <code>ConsumerConfig#GROUP_INSTANCE_ID_CONFIG</code> on the client side, stream app will crash.
```

Comment:
Do we need this?

Reply:
We need it for user to trouble shooting when they set group.instance.id on client where broker doesn't support.

Reply:
Yes, but as mentioned below, I would prefer to have this information is a proper "static group membership" sections. Having it in the upgrade guide does not seem to be the best place.